lib/commands/audit.js

```diff
@@ -37,6 +39,14 @@ class Audit extends ArboristWorkspaceCmd {
   }
 
   async exec (args) {
+    if (args[0] === 'signatures') {
```

Comment:
is this robust enough? like if i run `npm audit --production signatures` will it work?

Reply:
Ah yeah possibly not, will write some test for these cases 👍

Reply:
Yes it's fine. args is stripped of params

```
   async exec (args) {
+    console.log(args)
+    return
```

```
$ node . audit --production foo
npm WARN config production Use `--omit=dev` instead.
[ 'foo' ]
```